src/views/About/index.js

```diff
@@ -0,0 +1,151 @@
+import React, { Component } from 'react';
+
+import Typography from 'material-ui/Typography';
+import './about.css';
+
+export default class About extends Component {
+  render() {
+    return (
+      <div className="container page-title about-container about">
+        <p className="highlight">
+          <Typography type="Headline" gutterBottom>
+        Conceived and built at Gordon College, by students for students.
+         Everyone should be involved in something at College. Its
+        what makes College your own Experience.</Typography></p>
+        <div className="about-content">
+          <div>
+            <h4 className="titletext">
+              <Typography type="title" gutterBottom>
+            Gordon’s 360 Mobile Activities Platform: The Scottie Faire in
+             your hand.</Typography></h4>
+            <ul>
+              <Typography type="body2" gutterBottom>
+                <li>Mobile-friendly, responsive web portal</li>
+                <li>Promotional access to student Involvement and leadership</li>
+                <li>Academic session-based Campus Activities and Leadership Experiences</li>
+                <li>Subscriber email feeds</li>
+                <li>Membership affinity connections</li>
+                <li>Student leader and Advisor Activity self-management</li>
+                <li>Rosters and administrative records</li>
+                <li>Student Membership and Leadership Experience Transcript</li>
+              </Typography>
+            </ul>
+          </div>
+          <div>
+            <h4 className="titletext">
+              <Typography type="title" gutterBottom>
+                Institutional Benefits
+              </Typography></h4>
+            <ul>
+              <Typography type="body2" gutterBottom>
+                <li>In-house Student Record relationship</li>
+                <li>Official “Gordon Experience Transcript” with unofficial portal view</li>
+                <li>Academic Advising resource</li>
+                <li>Leadership and Involvement advising resource</li>
+                <li>Institutional reporting</li>
+              </Typography>
+            </ul>
+          </div>
+        </div>
+        <div className="about-content">
+          <h3 className="titletext">
+            <Typography type="title" gutterBottom>
+                Origins
+            </Typography></h3>
```

Comment:
It looks like this `</h3>` should be on its own line. ESLint picked it up when I opened the file in my editor. Can you look through the file and correct any other linting errors?

To double check, run `npm run lint` in your terminal to see if there are any linting errors you missed.